app/views/users/index.html.erb

```diff
@@ -6,11 +6,11 @@
       <li><%= merchant.street %></li>
       <li><%= merchant.created_at %></li>
         <% if current_admin? %>
-        <li><%= link_to(admin_merchants_path(merchant.id)) %></li>
+        <li><%= link_to("#{merchant.name}", admin_merchant_path(merchant.id)) %></li>
           <% if merchant.enabled %>
-          <li><%= button_to("disable") %></li>
+          <li><%= button_to "disable", "/deactivate", method: :enable, params: { user_id: merchant.id, enabled: false } %></li>
```

Comment:
Should probably be `deactivate_path`, it looks like you've got the routes worked out for that to work!